plugins/techdocs-backend/src/service/router.ts

```diff
@@ -112,8 +112,7 @@ export async function createRouter({
     const catalogRes = await fetch(`${catalogUrl}/entities/by-name/${triple}`);
     if (!catalogRes.ok) {
       const catalogResText = await catalogRes.text();
-      res.status(catalogRes.status);
-      res.send(catalogResText);
+      res.status(catalogRes.status).json(catalogResText);
```

Comment:
Hmm no this is not right, the code explicitly reads the remote content as text and then sends as text again, as a passthrough.

Reply:
reverted!